database/psql.sql

```diff
@@ -0,0 +1,47 @@
+-- terminal command to ingest data: psql -f database/psql.sql mydb
+-- generic: psql -f <file/to/ingest> <database_to_use>
+
+DROP TABLE IF EXISTS reviewers;
+
+CREATE TABLE reviewers (
+  id  BIGINT PRIMARY KEY,
+  firstName TEXT,
+  lastName TEXT,
+  color TEXT NOT NULL,
+  vip BOOLEAN NOT NULL
+);
+
+COPY reviewers(id, color, vip, firstName, lastName)
+FROM '/home/mrclean/HRR43/SDC/reviews/CSV/Reviewers.csv' DELIMITER ',' CSV HEADER;
+
+
+DROP TABLE IF EXISTS restaurants;
+
+CREATE TABLE restaurants (
+  id BIGINT PRIMARY KEY,
+  city TEXT NOT NULL
+);
+
+COPY restaurants (id, city)
+FROM '/home/mrclean/HRR43/SDC/reviews/CSV/Restaurants.csv' DELIMITER ',' CSV HEADER;
+
+DROP TABLE IF EXISTS reviews;
+
```

Comment:
lgtm. i did something similar. i had to change a few things myself on the data generation side because certain faker methods produce string with commas in them, which tripped up the CSV copy to postgres for me.